internal/performance/performance.go

```diff
@@ -136,15 +140,14 @@ func (c *PerfCollector) processEntityMetrics(metricsValues *types.PerfEntityMetr
 			c.logger.Debugf("vCenter returned no samples for the metric: %v", name)
 			continue
 		}
-
-		if len(metricValueSeries.Value) != 1 {
-			c.logger.Debugf("The metric: %v is not containing one sample, this is not expected", name)
-			continue
+		var metricVal int64
+		for _, val := range metricValueSeries.Value {
+			metricVal += val
```

Comment:
this seems a bit strange at first glance. summing metric values is not a common thing. can you add a test that exemplifies what type of inputs we have and what is the expected value?

Reply:
you're right , that should be an average , someway i forgot to include the division by length. I will include it

Reply:
I just check that that array should be len 1 because of the maxSample=1 , but instead the api is retriving multiples samples with the same values (for historical intervals), so, it doesn't make much sense to perform an average of that. I have check it's not 0 and just picked the first one.